winch/codegen/src/codegen/env.rs

```diff
@@ -32,17 +29,46 @@ pub struct TableData {
 /// A function callee.
 /// It categorizes how the callee should be treated
 /// when performing the call.
+#[derive(Clone)]
 pub enum Callee {
     /// Locally defined function.
     Local(CalleeInfo),
     /// Imported function.
     Import(CalleeInfo),
     /// Function reference.
     FuncRef(WasmFuncType),
+    /// A built-in function.
+    Builtin(BuiltinFunction),
+}
+
+impl Callee {
+    /// Get the built-in function metadata.
+    ///
+    /// # Panics
+    /// This function panics if the [`Callee`] is not a built-in function.
+    pub fn get_builtin(&self) -> &BuiltinFunction {
```

Comment:
Maybe name this `unwrap_builtin`?

Reply:
As I was fixing this I realized that this was actually dead code, so I ended up removing `get_builtin` and `get_info` entirely.